src/persistence/settingsserializer.cpp

```diff
@@ -412,11 +411,9 @@ void SettingsSerializer::readIni()
         // Add all keys
         if (!s.group().isEmpty())
             beginGroup(s.group());
+
         for (QString k : s.childKeys())
-        {
             setValue(k, s.value(k));
-            //qDebug() << "Read key "<<k<<" in group "<<group<<":\""<<groups[group]<<"\"";
-        }
```

Comment:
Why? Our coding style speaks, that `if / while / for / switch` with one line should be without brackets (https://github.com/tux3/qTox/blob/master/CONTRIBUTING.md#coding-style)